res/css/structures/_FilePanel.scss

```diff
@@ -109,7 +109,6 @@ limitations under the License.
     position: initial;
     font-size: $font-14px;
     opacity: 1.0;
-    color: $event-timestamp-color;
```

Comment:
This is redundant as specified with https://github.com/matrix-org/matrix-react-sdk/pull/8818/files#diff-ee6630d9ea69841b23949b052300770cf30db1afe7d269d4f4e769ef49f346b5R20.

|Before|After|
|---------|------|
|![filepanel_before](https://user-images.githubusercontent.com/3362943/173214449-1da452bf-da49-4cf9-9fba-8808eaaeec47.png)|![filepanel_after](https://user-images.githubusercontent.com/3362943/173214399-a276ba92-cb93-479f-b861-6db15ead8e31.png)|